LearningMissionLab/Schedule.cs

```diff
@@ -1,10 +1,32 @@
 ﻿using System;
+using System.Collections.Generic;
+
 namespace LearningMissionLab
 {
     public class Schedule
     {
-        public Schedule()
+        DateTime _startDate;
+        DateTime _finishDate;
+        DateTime _createDate;
+        DateTime _updateDate;
+        List<ScheduleEvent> _scheduleEventList;
+
+        public Schedule(DateTime startDate, DateTime finishDate, List<ScheduleEvent> scheduleEventList,
+                        DateTime createDate, DateTime updateDate)
```

Comment:
You are not using input parameters:  DateTime createDate, DateTime updateDate. Please fix. Consider using two constructors: (1) with DateTime createDate, DateTime updateDate and (2) without those parameters.